esp/src/src/nls/hpcc.ts

```diff
@@ -474,7 +474,21 @@ export = {
         Lock: "Lock",
         LogAccessType: "Log Access Type",
         LogDirectory: "Log Directory",
+        LogEventType: "Log Event Type",
         LogFile: "Log File",
+        LogFilterComponentsFilterTooltip: "Only include logs from the selected component/container",
```

Comment:
@rpastrana Just wanted to get you to look at the tooltip strings, if you had any ideas for the content